config/dev.exs

```diff
@@ -7,7 +7,14 @@ config :logflare,
 config :logflare, LogflareWeb.Endpoint,
   server: true,
   http: [
-    port: System.get_env("PORT") || 4000
+    port: System.get_env("PORT") || 4000,
+    thousand_island_options: [
+      transport_options: [
+        # https://stackoverflow.com/questions/41885427/use-so-reuseport-with-phoenix
+        # mac-specific flags
+        {:raw, 0xFFFF, 0x0200, <<1::32-native>>}
```

Comment:
Same here, why bother as we already have `{:reuseport, true}` set? Maybe you wanted `SO_REUSEADDR`, but that option is also exposed without using raw socket options.